packages/gatsby-source-contentful/src/download-contentful-assets.js

```diff
@@ -49,6 +49,12 @@ const downloadContentfulAssets = async gatsbyFunctions => {
         reporter.warn(`The asset with id: ${id}, contains no file.`)
         return Promise.resolve()
       }
+      if (!node.file.url) {
+        reporter.warn(
+          `The asset with id: ${id} has a file but the file contains no url.`
+        )
```

Comment:
This has potential to spam terminal output if we do it for every asset missing file. But maybe that's fine 🤔 

How can you end up in scenario where there is asset without file attached to it in Contentful?

Reply:
Oh I see, you mentioned this in the issue